universal-application-tool-0.0.1/app/services/cloud/aws/SimpleStorage.java

```diff
@@ -66,13 +66,14 @@ public SimpleStorage(
         });
   }
 
-  public URL getPresignedUrl(String key) {
+  @Override
+  public String getPresignedUrlString(String key) {
     // TODO(#1841): support storing and displaying original filenames for AWS uploads
-    return getPresignedUrl(key, Optional.empty());
+    return getPresignedUrlString(key, /* originalFileName= */ Optional.empty());
   }
 
   @Override
-  public URL getPresignedUrl(String key, Optional<String> originalFileName) {
```

Comment:
Curious, why remove the structure of URL rather than let the caller decide?

Reply:
See this comment: https://github.com/seattle-uat/civiform/pull/1833#discussion_r791157223
I think it's more about the fact that we're always converting to String anyways and java.net.URL has some tricky bugs